### PR TITLE
finished protobuf update

### LINKED ISSUE #131: Update Protobuf to Support new format
### Description

Update protobuf to ServerData v2.

### Acceptance Criteria

Data is read using Calypso simulate mode.

### Proposed Solution

Update the .proto file from the one found [here](https://github.com/Northeastern-Electric-Racing/Argos/blob/c944d04175319be0827955aff3032206d599678c/scylla-server/src/proto/serverdata.proto).  Update any API changes it makes, ensure you use correct timestamp units.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by mattrwang**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: NERODevelopment/src/controllers/configurationcontroller.cpp [hunk 1]
```diff
@@ -56,9 +56,12 @@ void ConfigurationController::upButtonPressed() {
...
     if (this->m_selectedConfigurationIndex == 3) {
-      this->m_model->sendMessage("driver", this->m_driverName);
-      this->m_model->sendMessage("system", this->m_systemName);
-      this->m_model->sendMessage("location", this->m_locationName);
+      // this->m_model->sendMessage("driver", this->m_driverName);
+      // this->m_model->sendMessage("system", this->m_systemName);
+      // this->m_model->sendMessage("location", this->m_locationName);
+      this->m_model->sendMessage("driver", 0);
+      this->m_model->sendMessage("system", 0);
+      this->m_model->sendMessage("location", 0);
```
Comment (jr1221) [inline NERODevelopment/src/controllers/configurationcontroller.cpp:64]: I assume this stuff going to get stripped out eventually?  I.e. no keyboard input.

File: NERODevelopment/src/models/mqtt_client.cpp [hunk 2]
```diff
@@ -85,10 +85,10 @@ void MqttClient::updateMessage(const QMqttMessage &msg) {
   // qDebug() << debug;
 }
 
-void MqttClient::sendMessage(const QString topic, const QString msg) {
-  serverdata::ServerData serverData;
+void MqttClient::sendMessage(const QString topic, const float value) {
+  serverdata::v2::ServerData serverData;
   serverData.setUnit("");
-  serverData.setValues({msg});
+  serverData.setValues({value});
```
Comment (jr1221) [inline NERODevelopment/src/models/mqtt_client.cpp:92]: can we add here a set time and give it the current microsecond since epoch system time?

**Review by jr1221 — CHANGES_REQUESTED** [review]
see comment about timing

**Comment by jr1221** [comment]
Tested and doesnt crash anymore.  dont know necessarily if it works bc we cant enter any screens.
